.github/workflows/ci.yml

```diff
@@ -174,9 +174,7 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - config-empty
```

Comment:
Why did you remove config-empty? it's used to test metastore only operations AFAIK. cc: @findepi